docs/stylesheets/extra.css

```diff
@@ -85,6 +85,19 @@
   display: inline; /* Show dark images in dark mode */
 }
 
+[data-md-color-scheme="astral-light"] img[src$="#only-light"],
+[data-md-color-scheme="astral-dark"] img[src$="#only-dark"] {
+  display: inline;
+  height: 250px;
+  width: 500px;
```

Comment:
Shouldn't these characteristics be set just for this specific image? Is there a reason you decided to implement it in the general css?

Reply:
That's a good suggestion. Initially I wanted to leave the index file unmodified, but I like your suggestion. Here it is.